src/engine/Settings.cs

```diff
@@ -349,7 +385,6 @@ public void ApplyAll(bool delayedApply = false)
             ApplySoundSettings();
         }
 
-        ApplyLanguageSettings();
```

Comment:
Removing this line breaks applying the language settings.
You can try it out if you change the language, save, close the game and open it. The new save won't be applied, but the drop down menu shows the selected language.

Reply:
You can also see the error by changing the language and pressing the reset button.